dissect/target/target.py

```diff
@@ -252,28 +254,23 @@ def open_all(cls, paths: list[Union[str, Path]], include_children: bool = False)
 
         def _find(find_path: Path):
             yield find_path
-
             if find_path.is_dir():
                 yield from find_path.iterdir()
 
-        loaded = False
+        at_least_one_loaded = False
 
         # Treat every path as a unique target spec
         for path in paths:
+            loaded = False
             path, parsed_path = extract_path_info(path)
 
-            found_loader = False
-
             # Search for targets one directory deep
             for entry in _find(path):
-                loader_cls = loader.find_loader(entry, parsed_path=parsed_path)
-
+                loader_cls = loader.find_loader(entry, parsed_path=parsed_path, fallbacks=[DirLoader, RawLoader])
                 if not loader_cls:
                     continue
 
-                getlogger(entry).debug("Found compatible loader: %s", loader_cls)
-                found_loader = True
-
+                getlogger(entry).debug("Attempting to use loader: %s for: %s", loader_cls, entry)
```

Comment:
Question: doesn't `getlogger(entry)` already make the log lines easily findable for that entry (path)?